File: src/test/resources/org/springframework/data/hadoop/scripting/basic-script.py
from java.util import UUID
from org.apache.hadoop.fs import Path

print "Home dir is " + str(fs.homeDirectory)
print "Work dir is " + str(fs.workingDirectory)
print "/user exists " + str(fs.exists("/user"))

name = UUID.randomUUID().toString()
scriptName = "src/test/resources/test.properties"
fs.copyFromLocalFile(scriptName, name)
print Path(name).makeQualified(fs)
fs.getLength(name)

File: src/test/resources/org/springframework/data/hadoop/scripting/basic-script.rb
require 'java'
include org.apache.hadoop.fs

puts "Home dir is " + $fs.homeDirectory.to_s
puts "Work dir is " + $fs.workingDirectory.to_s
puts "/user exists " + $fs.exists("/user").to_s

name = java.util.UUID.randomUUID().to_s
scriptName = "src/test/resources/test.properties"
$fs.copyFromLocalFile(scriptName, name)
puts Path.new(name).makeQualified($fs)
# return the file length 
$fs.getLength(name)